src/agent/rustjail/src/validator.rs

```diff
@@ -82,6 +83,17 @@ fn hostname(oci: &Spec) -> Result<()> {
 
 fn security(oci: &Spec) -> Result<()> {
     let linux = get_linux(oci)?;
+    let label_pattern = r".*_u:.*_r:.*_t:s[0-9]|1[0-5].*";
+    let label_regex = Regex::new(label_pattern)?;
+
+    if let Some(ref process) = oci.process {
+        if !process.selinux_label.is_empty() && !label_regex.is_match(&process.selinux_label) {
+            return Err(einval());
```

Comment:
Suggestion adds more detailed message in error message than only one error code.

Reply:
Would suggest the same for the remaining error messages as well.

Reply:
Thank you.  I fixed them with detailed error messages.